### PR TITLE
[all-permissions] Support Firefox.

### PR DESCRIPTION
**Pull request by lgarron**
Also added the ms prefixed functions for good measure (even though IE doesn't support most of them).

### CODE REVIEW

File: demos/all-permissions.html [hunk 1]
```diff
@@ -76,6 +76,12 @@
   }
 };
 
+navigator.getUserMedia           = (    navigator.getUserMedia       ||     navigator.webkitGetUserMedia       ||     navigator.mozGetUserMedia       ||     navigator.msGetUserMedia      );
```
Comment (adrifelt) [inline demos/all-permissions.html]: do ur lines even wrap bro

Comment (lgarron) [inline demos/all-permissions.html]: Yeah, I originally wrapped, but it was obnoxiously many lines. But fine. :-P

**Comment by lgarron** [comment]
Updated with wrapping.

File: demos/all-permissions.html (hunk 1 above)
Comment (adrifelt) [inline demos/all-permissions.html]: this is nice looking now :)

**Comment by adrifelt** [comment]
lgtm :+1:

**Comment by lgarron** [comment]
out

**Pull request merged**